tests/integration/specs/common/autostore.spec.js

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+import nock from 'nock';
+import { init, DataStore, DataStoreType, User, Query } from '__SDK__';
+import { collectionName } from '../config';
+import { cleanUpAppData } from '../utils';
+
+describe('AutoStore', function() {
+  const cachedDocs = [];
+
+  before(function () {
+    // Turn off nock
+    return nock.restore();
+  });
+
+  before(function() {
+    // Initialize the SDK
+    return init({
+      appKey: process.env.APP_KEY,
+      appSecret: process.env.APP_SECRET
+    });
+  });
```

Comment:
It might be a good idea if we added the option to run the tests for various offline storage providers. I use the utils.setOfflineProvider() function in other suites:
```
before(() => {
      const initProperties = {
        appKey: process.env.APP_KEY,
        appSecret: process.env.APP_SECRET,
        masterSecret: process.env.MASTER_SECRET
      }
      return Kinvey.init(utilities.setOfflineProvider(initProperties, process.env.OFFLINE_STORAGE));
    });
```

Reply:
This should be an enhancement we add to the tests